questions/qa-validator-charset-check.en.html

```diff
@@ -63,6 +63,11 @@ <h2 class="notoc">Related links</h2>
     <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
     </div>
     
+    <div class="note" style="clear: none;">
+      <p><strong>NOTE!</strong> This is a historical article. Nowadays, you can go to the <a href="https://validator.w3.org/i18n-checker/">Internationalization (i18n) Checker</a> and it will tell you what your encoding is according to your declaration.
```

Comment:
```suggestion
      <p><strong>NOTE!</strong> This is a historical article. Nowadays, you can go to the <a href="https://validator.w3.org/i18n-checker/">Internationalization (i18n) Checker</a> and it will tell you what the encoding is according to the declaration at the top of the file.
```